core/src/storage/device.py

```diff
@@ -34,9 +34,15 @@
 _SLIP39_ITERATION_EXPONENT = const(0x11)  # int
 _SD_SALT_AUTH_KEY          = const(0x12)  # bytes
 INITIALIZED                = const(0x13)  # bool (0x01 or empty)
-_UNSAFE_PROMPTS_ALLOWED    = const(0x14)  # bool (0x01 or empty)
+_SAFETY_CHECK_LEVEL        = const(0x14)  # int
 
 _DEFAULT_BACKUP_TYPE       = BackupType.Bip39
+
+SAFETY_CHECK_LEVEL_STRICT   = const(0)
+SAFETY_CHECK_LEVEL_PROMPT   = const(1)
+_DEFAULT_SAFETY_CHECK_LEVEL = SAFETY_CHECK_LEVEL_STRICT
+if False:
+    StorageSafetyCheckLevel = int
```

Comment:
Probably useless. Seems that `const` and `Literal` don't work well together.

Reply:
wouldn't `SAFETY_CHECK_LEVEL_PROMPT = const(1)  # type: Literal[1]` work?

Reply:
That leads to: _error: Incompatible types in assignment (expression has type "int", variable has type "Literal[0]")  [assignment]_

Reply:
hm.
let's just use `int` then

Reply:
wait
this could be fixed in `micropython.pyi`. let me try something...

Reply:
this should work:
```
from typing import TypeVar

T = TypeVar("T", bound=int)

def const(c: T) -> T: ...
```

OTOH i'm really not sure if any of this is worth it, using the `int` type in `safety_check_level`/`set_safety_check_level` is perfectly OK

Reply:
Agreed that it's not that useful in this case, though we can get advantage of it in the future so I added it in db3bc96 + 77333fc.